zerver/webhooks/travis/doc.md

```diff
@@ -0,0 +1,25 @@
+See your Travis CI build notifications in Zulip!
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+Finally, set up a webhook in your `.travis.yml` file:
+
+```
+notifications:
+webhooks:
+- {{ external_api_uri_subdomain }}/v1/external/travis?stream=travis&amp;topic=build-status&amp;api_key=abcdefgh
+```
+
+By default, pull request events are ignored since most people
+don't want notifications for new pushes to pull requests.  To
+enable notifications for pull request builds, just
+add `&ignore_pull_requests=false` at the end of the url.
+
+
+**Congratulations! You're done!**
+
+When a code build occurs,
+you will receive a notification on the specified stream and topic.
+![](/static/images/integrations/travis/001.png)
```

Comment:
I would recommend moving line 23 and 24 to line 20 and using the `{congrats.md}` macro, like so.
~~~
When a code build occurs, you will receive a notification on the specified
stream and topic.

{!congrats.md!}

![](/static/images/integrations/travis/001.png)
~~~